python-impl/impl-test.py

```diff
@@ -4,6 +4,7 @@
 
 from fields import Fq, Fq2, Fq6, Fq12
 from ec import (
+    JacobianPoint,
```

Comment:
We should add these tests to the github actions so that we're testing the python implementation on every build too. Happy to do that for you if you'd like.

Reply:
It takes like 10 minutes to run the python implementation tests, do you think it is worth it?

Reply:
An option would be to make it an "on click" action. It's there on every run but doesn't run unless you go over to actions and click Run. Thoughts?